### ISSUE TITLE
"announce-list" paramtert should not be added automatically

### ISSUE THREAD

**Issue by eversins** [drizt/torrent-file-editor#79]
If you load a torrent into torrent-file-editor and save without changes the "announce-list" parameter gets added automatically with the same value as "announce".

This parameter can cause errors on some trackers, took me a while to realize t-f-e was the cause.

"announce-list" should not be added automatically, maybe make it a toggle?

Thank you for your work on this editor!

**Comment by eversins** [comment]
Thank you for your quick fix. Unfortunately the "announce-list" parameter still gets added if you change the announce tracker.

**Comment by drizt** [comment]
I don't build new version yet.

**Comment by eversins** [comment]
Thank you for your reply.
I am using: v0.3.13-3-gda74eba (Wednesday, 21. November 2018)

announce-list still gets added or am I missing something? Thanks.

**Comment by drizt** [comment]
Do you change tracker?

**Comment by eversins** [comment]
Yes, if you change the tracker or add a tracker to a torrent which has none, "announce-list" gets added automatically. 
Where only "announce" should be added because some trackers can't handle "announce-list".

**Comment by drizt** [comment]
In current implememantation I only disable automatic `announce-list` adding when `announce` is not changed.

**Comment by drizt** [comment]
You can edit `announce` in **Tree** tab. In this case `announce-list` won't be added.

**Comment by eversins** [comment]
Awesome thank you.

I don't know enough about the torrent standard, so please excuse my ignorance. I assume announce-list is used once you need more than one tracker? Maybe a simple if announce >1 use announce-list else just use announce would be a simple "fix".

For me personally the current solution is absolutely sufficient. Thank you.
